install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -28,7 +28,14 @@ global:
     accessLogFile: "/dev/stdout"
 
     # If set, newly injected sidecars will have core dumps enabled.
-    enableCoreDump: false
+    # Deprecated, use maxNumCoreDumps below.
+    enableCoreDump: true
```

Comment:
Can't we just remove it if it is deprecated? It is just an install option, not an actual field in an API.

Reply:
One could argue the helm install options are the de-facto API for system deployment config.

Reply:
+1, helm install options are de-factor API for config and critical for upgrade, should be treated as an API.

Reply:
Also there is no clear reason for deprecation.

Reply:
We should not have 2 options to enable core files, this is why it is confusing.

I am curious, what happens actually when a helm option is removed? Why would that cause problems for the upgrade? The yaml files will be generated according to the new options, and that's it. There is no API here.